sdk/storage/azure-storage-blob/tests/test_retry.py

```diff
@@ -434,6 +438,25 @@ def test_invalid_account_key(self, storage_account_name, storage_account_key):
         # No retry should be performed since the signing error is fatal
         self.assertEqual(retry_counter.count, 0)
 
-
-# ------------------------------------------------------------------------------
-
+    @pytest.mark.live_test_only
```

Comment:
Does this need to be Live only?

Reply:
I think so? I'm really not sure but when I run it without the decorator, the test gets stuck. Maybe I could mock the transport? I'm open to suggestions.

Reply:
I don't see anything here that would cause a problem recording since you're mocking the response streaming. Did you try generating a recording?

- Remove the decorator, set `RUN_IN_LIVE` to `True`, and set `SKIP_LIVE_RECORDING` to `False` in the settings file.
- Run the test. It should generate a `.yaml` file.
- Then set `RUN_IN_LIVE` to `False` and run the test again.

Reply:
@mccoyp to the rescue if necessary